js/start-game.js

```diff
@@ -13,3 +13,13 @@ document.querySelector('#splash-open-game').addEventListener('click', function()
     listenCityClick()
     listenKeypress()
 })
+
+document.querySelector('#instructions-tab').addEventListener('click', function() {
```

Comment:
no active state on these tabs, need to show the users which one they are on

Reply:
changed in 22106c3